src/jumpstart/useRightJumpstartScreen.ts

```diff
@@ -0,0 +1,11 @@
+import { useSelector } from 'react-redux'
+import { jumpstartIntroHasBeenSeenSelector } from 'src/jumpstart/selectors'
+import { Screens } from 'src/navigator/Screens'
+import { jumpstartSendTokensSelector } from 'src/tokens/selectors'
+
+export function useRightJumpstartScreen() {
```

Comment:
@kathaypacific With introduction of intro screen for multiple flows (previously it was only for one) I figured Intro screen can be a good starting point for all the scenarios. 
My only concern is that we now have three scenarios, 2 of which start on the intro screen and one skips it. I definitely agree that considering the points you've described, with this implementation it makes it unnecessary complex to manage navigation for jumpstart in general. But maybe we could change the main screen from "Enter Amount" to Intro screen and manage all the flows from there? 
- it it needs to be shown - it will render whatever is necessary based on the user's assets 
- if it needs to be hidden - it can replace itself with "Enter Amount"

Having "Enter Amount" a main screen seems to be a bit confusing to me as it's just a part of certain flows, not all of them. But maybe I'm thinking about it wrong?

Also, agree on the naming thing, I just couldn't figure out better way to call it without wasting too much time on it. Although, I agree that coming to the point of needing to name things like this is kinda a marker of somewhat weird implementation.